python/common/org/python/types/Object.java

```diff
@@ -362,12 +362,17 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
                 return false;
             }
         }
-
+        
```

Comment:
Minor code style point: could you remove the added leading whitespace here? And below, the `if (...){` should have a space before the left curly brace, so that it's `if (...) {`.